README.md

```diff
@@ -92,5 +91,5 @@ That pushes the necessary files to the `gh-pages` branch.
 ##  Running Tests
 
 ```shell
-$ python manage.py test
+tox
```

Comment:
Should we say something about installing tox first ([example](https://github.com/cfpb/django-college-costs-comparison#testing))?